vexim/login.php

```diff
@@ -59,39 +60,43 @@
   print $cryptedpass . "<br>\n";
   */
 
-	# if they have the wrong password bail out
-	if ($cryptedpass !== $row['crypt']) {
-		header ('Location: index.php?login=failed');
-		die();
-	}
+  # if they have the wrong password bail out
+  if ($cryptedpass !== $row['crypt']) {
+    header ('Location: index.php?login=failed');
+    die();
+  }
 
-	# populate session variables from what was retrieved from the database (NOT what they posted)
-    $_SESSION['localpart'] = $row['localpart'];
-    $_SESSION['domain'] = $row['domain'];
-    $_SESSION['domain_id'] = $row['domain_id'];
-	$_SESSION['crypt'] = $row['crypt'];
-    $_SESSION['user_id'] = $row['user_id'];
+  # populate session variables from what was retrieved from the database (NOT what they posted)
+  $_SESSION['localpart'] = $row['localpart'];
+  $_SESSION['domain'] = $row['domain'];
+  $_SESSION['domain_id'] = $row['domain_id'];
+  $_SESSION['crypt'] = $row['crypt'];
+  $_SESSION['user_id'] = $row['user_id'];
 
-	# redirect the user to the correct starting page
-	if (($row['admin'] == '1') && ($row['type'] == 'site')) {
-                if($_POST['crypt']=="CHANGE") {
-                  header ('Location: sitepassword.php');
-                  die();
-                }
-		header ('Location: site.php');
-		die();
-	} 
-	if ($row['admin'] == '1') {
-		header ('Location: admin.php');
-		die();
+  # redirect the user to the correct starting page
+  if (($row['admin'] === '1') && ($row['type'] === 'site')) {
+    if($_POST['crypt'] === "CHANGE") {
+      header ('Location: sitepassword.php');
+      die();
     }
-	if (($row['domainenabled'] == '0')) {
-		header ('Location: index.php?domaindisabled');
-		die();
-}
-	
-	# must be a user, send them to edit their own details
-	header ('Location: userchange.php');
-	
+    header ('Location: site.php');
+    die();
+  }
+  if ($row['admin'] === '1') {
+    header ('Location: admin.php');
+    die();
+ }
+ if (($row['userenabled'] === '0')) {
+   header ('Location: index.php');
```

Comment:
But that is also true for disabled domains.
Should we even set the session variables in these cases?